include/Macro.h

```diff
@@ -883,6 +883,9 @@
 #  define ATAN2( a, b )    atan2f( a, b )
 #endif
 
+// stringizing operator in C/C++
+#define STR( x )            #x
+#define SHOW_MACRO( x )     STR( x )
```

Comment:
Use the existing macros `QUOTE` and `EXPAND_AND_QUOTE` defined [here](https://github.com/gamer-project/gamer/blob/master/include/Macro.h#L915-L916).